### DISCUSSION TITLE
如果您遇到支付困难时可以使用这链接付费, 同时也可以用支付宝!

### DISCUSSION THREAD

**Discussion by MaxMKMax** [getlantern/lantern-forum-cn#481] [Announcements]
支付宝在修复

**Comment by Lpf777** [comment]
付款后却不能用，在哪里可以申请退款啊

**Reply by Lpf777** [reply]
通过你提供的支付宝链接付款了一年，可是登陆蓝灯仍旧显示未付款！请问我通过你提供的链接打款去哪里了？为什么蓝灯仍旧用不了？请退款！

**Reply by MaxMKMax** [reply]
@Lpf777 您好. 请您提供我支付单据和您的账号邮箱. 我帮您查看. 请给我们几天的时间. 肯定帮您解决问题! 谢谢.

**Reply by MaxMKMax** [reply]
@Jianliang-ouc 
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
您好. 为了保护您的隐私. 请您发支付单据和账号邮箱到这个链接. 谢谢.

**Reply by Lpf777** [reply]
谢谢你的耐心服务，通过支付宝已成功退款到账

**Reply by Lpf777** [reply]
收到你发的退款链接时，那时蓝灯还不能使用。提交申请退款的当天晚上恰好能用了，但是支付宝退款成功后就用不了了。于是在蓝灯官网上用visa信用卡重新支付了一年费用，但是可能因为电脑问题，蓝灯软件一直闪退用不了，所以再次申请退款，但是信用卡一直没有收到退款，能帮忙处理一下吗？

**Comment by henrylyubj** [comment]
支付宝用不了：“支付失败，商家收款账户状态异常。请提醒商家检查收款账户信息，或使用其他方式付款”

**Reply by MaxMKMax** [reply]
@henrylyubj 嗯嗯. 我们暂时关掉支付宝支付系统因为有一些问题. 请您是看用别的方法支付可以吗? 谢谢您.

**Comment by henrylyubj** [comment]
实在没有别的支付方式，银联，支付宝，也就这两种

Mike K ***@***.***> 于2024年6月18日周二 00:01写道：

**Reply by MaxMKMax** [reply]
@henrylyubj 
我在另一个帖子回复您了. 请查看.

**Comment by yangziyun-dot** [comment]
银行卡被拒绝，如何解决？

**Reply by MaxMKMax** [reply]
@yangziyun-dot 您好. 您可以试试看微信支付. 或者用别的银行卡. 谢谢您支持蓝灯.

**Comment by Lpf777** [comment]
已经和银行沟通过。客服说您那边原路退款时请选择银联渠道，不是VISA渠道。7月1日我付款的时候是用人民币付款，不是美金付款。请再次操作退款看看。



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年7月6日(周六) 上午9:45
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 如果您遇到支付困难时可以使用这链接付费, 同时也可以用支付宝! (Discussion #481)




 
银行卡被拒绝，如何解决？

**Reply by MaxMKMax** [reply]
@Lpf777 您好. 我这里是没权限操作退款的.
我们会帮您解决问题. 请您放心. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱. 
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ

*备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!

**Reply by Lpf777** [reply]
通过这个链接提交申请退款就可以了吗？

**Reply by MaxMKMax** [reply]
@Lpf777 对的. 在这个链接提交问题. 如果过了 2 周您的问题没解决再回来找我. 我帮您直接对负责部门.

**Comment by 279577187** [comment]
@Mike777Khoo 请问，使用一年期支付宝付款后，多久可以生效？

**Reply by Lpf777** [reply]
请问你的蓝灯软件会闪退吗

**Reply by Lpf777** [reply]
上次我用支付宝交费成功后大概一个星期可以用

**Reply by MaxMKMax** [reply]
@279577187 一般付费后您会收到激活码. 只要能够收到邮件就能马上使用了. 但如果您无法收到邮件, 就要来找我帮忙. 我会帮您提交问题然后一般 2-5 天就能解决问题并使用.

**Reply by MaxMKMax** [reply]
@279577187 对了, 您可以帮我个忙吗? 您在付费的选项里, 帮我查看有支付宝的选项吗? 我们在做测试. 想了解用户可以看到吗. 谢谢您!

**Comment by liangzi16** [comment]
@Mike777Khoo,你好， 我是2024年7月2日付款，一直未收到激活码，邮箱[39053595@qq.com](mailto:39053595@qq.com),付款截图见图，邮箱里垃圾箱、收件箱都已经查看过，一直未收到激活码，麻烦帮处理退款或者发送激活码，谢谢！
![微信图片_20240715122308](https://github.com/user-attachments/assets/f8f88f56-c248-4e40-b4b1-d995220975ec)

**Reply by MaxMKMax** [reply]
@liangzi16 您好. 已经帮您提交问题了. 会给您提供激活码. 这两天请查看邮箱或这个帖子. 谢谢.

**Reply by MaxMKMax** [reply]
@liangzi16 
您的专属激活码:
MTFHJ-FWJ6X-V976T-PTBRG-K2GM7

**Reply by liangzi16** [reply]
@Mike777Khoo,你好，谢谢之前发我的激活码。我购买的年卡，但是今天显示我的专业版已经过期，距离你发我激活码刚好一个月，应该是时间弄错了，麻烦时间是一年的新激活码发我下，附件是我的付款记录和现在蓝灯显示过期的截图，我的邮箱是39053595@qq.com，谢谢！
![微信图片_20240816103138](https://github.com/user-attachments/assets/b4d4571f-133d-4ad3-a93d-1250c215f93b)
![微信图片_20240816103130](https://github.com/user-attachments/assets/98df6510-92da-4f4a-ba39-68abc47ce2b2)

**Comment by MaxMKMax** [comment]
@liangzi16 您好. 已经帮您提交问题了. 会给您提供激活码. 这两天请查看邮箱或这个帖子. 谢谢.

**Comment by evilmanaya** [comment]
@Mike777Khoo 
你好， 我是2024年7月20日付款，一直未收到激活码，邮箱evilmanaya@gmail.com,付款截图见图，邮箱里垃圾箱、收件箱都已经查看过，一直未收到激活码，麻烦帮处理退款或者发送激活码，谢谢！
![fd3677b5284517385e6ffce45da17f0](https://github.com/user-attachments/assets/76024643-d988-4c5f-ae23-76ec992cc6f8)

**Reply by MaxMKMax** [reply]
@evilmanaya 您好. 已经帮您提交问题了. 会给您提供激活码. 这两天请查看邮箱或这个帖子. 谢谢.

**Reply by evilmanaya** [reply]
@Mike777Khoo 我邮箱还是没激活码，能帮忙尽快处理么，好多天了

**Reply by MaxMKMax** [reply]
@evilmanaya 
您好. 这是您的激活码:
BHFD3-GDYF7-MYVTD-9P4QG-32MJR

**Comment by ShawnAlv** [comment]
请问链接在哪里？好像没有看到，我手机app上付不了款，一直提示卡被拒绝，也没找到支付宝或者微信付款的地方

**Reply by MaxMKMax** [reply]
@ShawnAlv 对了, 您可以帮我个忙吗? 您在付费的选项里, 帮我查看有支付宝的选项吗? 我们在做测试. 想了解用户可以看到吗. 谢谢您!

**Comment by Coasenorth** [comment]
您好，已经通过支付宝付费，但是手机APP迟迟没有显示完成续期，还是旧的时间期限。以往都是付费成功后马上更新，现在大约需要多久，如何能解决呢？

**Reply by Coasenorth** [reply]
@Mike777Khoo 您好，我的邮箱是htifly@126.com，请帮忙查一下

**Reply by MaxMKMax** [reply]
@Coasenorth 您好. 请您提供我支付单据我帮您提交问题. 同时想请问一下, 您今天是可以看到支付宝选项对吗?
除了您形容的问题, 还有什么别的问题吗?

**Reply by Coasenorth** [reply]
![560644153](https://github.com/user-attachments/assets/44b34157-aac9-4b7a-b4f6-ed26a9c64fb8)

**Reply by Coasenorth** [reply]
@Mike777Khoo 这个是单据，续费两年的，帮忙查一下。谢谢

**Comment by MaxMKMax** [comment]
对了, 各位可以帮我个忙吗? 您在付费的选项里, 帮我查看有支付宝的选项吗? 我们在做测试. 想了解用户可以看到吗. 谢谢您!

同时您如果在Alipay付费后遇到什么问题请在这个帖子反馈. 我们会帮忙解决问题. 谢谢!

**Reply by Coasenorth** [reply]
手机的app（安卓）中可以看到支付宝选项，桌面版的看不到。

**Comment by cntceo** [comment]
账号 469323024@qq.com 
已经支付了，却收不到验证码，激活不了还是免费版，是否支持重新绑定邮箱cntceo@gmail.com
![Uploading WechatIMG502.jpg…]()

**Reply by cntceo** [reply]
![WechatIMG502](https://github.com/user-attachments/assets/7c2e5b2a-909b-46f9-8f9f-cde51d7047fd)

**Comment by wee99999** [comment]
现在不能续费吗，一直转圈，

**Reply by MaxMKMax** [reply]
@wee99999 您好. 请问您可以提供我们截图吗? 那个支付页面. 请问您支付不了的问题已经解决了吗?
